### ISSUE TITLE
Preview doesn't update when pressing DELETE for a single character

### ISSUE THREAD

**Issue by Kage-Yami** [daiyy2009/quick-preview#1]
Looks like using the `DELETE` key to remove a single character doesn't update the side-by-side preview. `BACKSPACE` works as expected, as does `DELETE` when multiple characters are selected.

### Reproduction steps:

``` html
<!doctype html>

<html>
    <head>
        <meta charset="utf-8">
        <title>Example Page</title>
    </head>

    <body>
        <h1>Example Page</h1>
    </body>
</html>
```

1. Open the side-by-side preview with `CTRL+SHIFT+P -> Side Preview`
2. Place the cursor just before "Page" in the `h1` element (line 10).
3. Press the `DELETE` key.

### Expected outcome:

The "P" for "Page" is removed from the side-by-side preview.

### Actual outcome:

The side-by-side preview is not updated.

**Comment by daiyy2009** [comment]
@Kage-Yami 
Thank you for your comment.

Now i have used ```vscode.workspace.onDidChangeTextDocument``` instead of ```vscode.window.onDidChangeTextEditorSelection```, the second doesn't triggered when press ```Del``` key. 

I have published version ```0.3.0``` to the market.

**Comment by daiyy2009** [comment]
Long time not get answered, close it.
